ci-operator/jobs/open-cluster-management/application-ui/open-cluster-management-application-ui-main-presubmits.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-open-cluster-management-application-ui-main-e2e-cluster
     rerun_command: /test e2e-cluster
+    skip_if_only_changed: ^docs/|\.(md|adoc)$|^(README|LICENSE)$
```

Comment:
Did we want to include OWNERS here?

Reply:
Probably. Let me get it passing all the tests first.